hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1518,6 +1518,16 @@
       to the OM.
     </description>
   </property>
+
+  <property>
+    <name>ozone.om.snapshot.rocksdb.metrics.enabled</name>
+    <value>true</value>
```

Comment:
Should we disable it but default and enable it on demand? Because when snapshot size grows, it will background services like SnapshotDeletingService.

Reply:
Yes, that sounds good.